RockPaperScissors/RockPaperScissors.cs

```diff
@@ -6,20 +6,44 @@ class Program
     {
         public static void Main()
         {
-            Console.WriteLine("Enter hand 1:");
+            Console.WriteLine("Let's play rock, paper, scissors!");
+            Console.WriteLine("Enter r, p, or s");
             string hand1 = Console.ReadLine().ToLower();
-            Console.WriteLine("Enter hand 2:");
-            string hand2 = Console.ReadLine().ToLower();
-            Console.WriteLine(CompareHands(hand1, hand2));
+            Console.WriteLine("The computer is choosing...");
+            string newChoice = computerChoice();
+            Console.WriteLine(newChoice);
+            Console.WriteLine(CompareHands(hand1, newChoice));
+
+        }
+
+
+        public static string computerChoice()
+        {
+            Random generator = new Random();
+            int randomNumber = generator.Next(0, 3);
```

Comment:
there is no reason to convert the randomNumber to a string, you can just compare it directly to the numbers 0,1 or 2.